transport/cloudservice.go

```diff
@@ -0,0 +1,173 @@
+package microservicetransport
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/LUSHDigital/microservice-core-golang/response"
+	"github.com/LUSHDigital/microservice-transport-golang/config"
```

Comment:
I guess your editor borked the import path (and in a few other places) here and continued using the old path

Reply:
fixed